src/main/scala/org/scalatest/xml/XmlMatchers.scala

```diff
@@ -28,7 +28,7 @@ trait XmlMatchers {
       val res = if(ignoreWhitespace) (e =#= a).successful else (e =?= a).successful
       MatchResult(
         res,
-        s"""$e isn't equal to $a""",
+        s"""$a was not equal to $e""",
```

Comment:
I also changed this to past tense to be consistent with built-in ScalaTest matchers.

(I think present tense is correct but consistency makes things easier.)

Reply:
Nice! Agree with you. Thanks